docs/conceptual/node/tx-pool.md

```diff
@@ -2,12 +2,80 @@
 title: Transaction Pool
 ---
 
-* What is the Transaction Pool?
 
-* What is priority?
+The transaction pool contains all transactions (signed and unsigned) broadcasted to the network that have been *received* and *validated* by the local node. The pool is responsible for:
 
-* What is dependency?
 
-* Graph VS Queue
+1) Checking if transactions are valid. `validity` of the transaction is not hard wired to the transaction pool, but is defined by the runtime through the `validate_transaction` function ([code](https://github.com/paritytech/substrate/blob/20a9b15cdbed4bf962a4447e8bfb812f766f2fbc/frame/executive/src/lib.rs#L304)). Example validity checks are:
 
-* What is the lifecycle of a transaction?
+* Transaction Index (nonce) correctness.
+* The account has enough funds to pay for the associated fees.
+* The signature is valid.
+
+2) If the transaction is valid, sorting the transactions into 2 transaction queues:
+   - **Ready Queue** -  Contains transactions that can be included in a new pending block and must follow the *exact* order in the ready queue.
+
+   - **Future Queue** - Contains transactions that *seem* valid so far, but the full node has yet to see the remaining transactions it depends on. For example, *Transaction B* is in the future queue and can only be included in a block after *Transaction A* has been included. However, the node is yet to see *Transaction A* in its transaction pool.
+
+Full nodes in the network all have their own local transaction pool and transaction queues. Therefore, full nodes may have a different view of the queues and transactions in the network.
```

Comment:
What's the difference between the transaction pool and the transaction queue? We've been using them interchangeably so it's confusing to show up like this.

Reply:
I've removed the paragraph